### ISSUE TITLE
Can't open context menu

### ISSUE THREAD

**Issue by Juerd** [gnome-terminator/terminator#796]
**Describe the bug**
The context menu does not appear when right-clicking a terminal.

**To Reproduce**
Right-click a terminal with my config.

**Expected behavior**
The context menu would appear when right-clicking a terminal.

**Screenshots**
The following error message is logged when right-clicking in a terminal, if I start terminator from an xterm:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal.py", line 1031, in on_buttonpress
    right_click[0](*right_click[1])
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal.py", line 1077, in popup_menu
    menu.show(widget, event)
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal_popup_menu.py", line 304, in show
    self.add_layout_launcher(menu)
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal_popup_menu.py", line 329, in add_layout_launcher
    item = self.menu_item(Gtk.MenuItem, 'layout_launcher', '_Layouts...')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal_popup_menu.py", line 55, in menu_item
    mask    = self.get_menu_item_mask(maskstr)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/terminatorlib/terminal_popup_menu.py", line 33, in get_menu_item_mask
    maskstr = maskstr.lower()
              ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lower'
```

**Desktop (please complete the following information):**
 - OS or Linux Distribution: Debian
 - Version: bookworm (12) + sid (unstable)
 - Display Technology: X11

**Additional context**
The bug arose after upgrading my Debian installation. The version of Terminator after upgrading is 2.1.3 (Debian package version 2.1.3-1); I'm not sure what the version was before upgrading, but I think it was 2.1.0 (Debian package version 2.1.0-2).

> Before opening an issue, please try starting terminator with an empty config file, by issuing `terminator -u -g /dev/null` and see if it starts normally.

With an empty config, it works as expected.

> If it does, and you still want to open the bug, then please attach the config file, usually found at `$HOME/.config/terminator/config` to the issue

<details>
<summary>.config/terminator/config</summary>

```
[global_config]
  focus = mouse
  handle_size = 2
  window_state = maximise
  inactive_color_offset = 1.0
  enabled_plugins = LaunchpadCodeURLHandler, APTURLHandler, LaunchpadBugURLHandler
  suppress_multiple_term_dialog = True
  title_use_system_font = False
  title_font = Sans 7
[keybindings]
  broadcast_all = <Primary><Shift><Alt>a
  edit_window_title = <Shift><Alt>t
  layout_launcher = None
[profiles]
  [[default]]
    visible_bell = True
    cursor_color = "#f3f3f3"
    font = DejaVu Sans Mono 9.5
    foreground_color = "#ffffff"
    scrollbar_position = hidden
    scrollback_lines = 5000
    palette = "#000000:#cd0000:#00cd00:#cdcd00:#4682b4:#cd00cd:#00cdcd:#e5e5e5:#4c4c4c:#ff0000:#00ff00:#ffff00:#1e90ff:#ff00ff:#00ffff:#ffffff"
    use_system_font = False
    bold_is_bright = True
  [[single]]
    visible_bell = True
    cursor_color = "#f3f3f3"
    font = DejaVu Sans Mono 9.5
    foreground_color = "#ffffff"
    scrollbar_position = hidden
    scrollback_lines = 5000
    palette = "#000000:#cd0000:#00cd00:#cdcd00:#4682b4:#cd00cd:#00cdcd:#e5e5e5:#4c4c4c:#ff0000:#00ff00:#ffff00:#1e90ff:#ff00ff:#00ffff:#ffffff"
    use_system_font = False
    bold_is_bright = True
    show_titlebar = False
[layouts]
  [[default]]
    [[[child0]]]
      order = 0
      parent = ""
      position = 0:25
      size = 2560, 1415
      type = Window
    [[[child1]]]
      order = 0
      parent = child0
      position = 707
      type = VPaned
    [[[child2]]]
      order = 0
      parent = child1
      position = 1280
      type = HPaned
    [[[child5]]]
      order = 1
      parent = child1
      position = 1280
      type = HPaned
    [[[terminal3]]]
      order = 0
      parent = child2
      profile = default
      type = Terminal
    [[[terminal4]]]
      order = 1
      parent = child2
      profile = default
      type = Terminal
    [[[terminal6]]]
      order = 0
      parent = child5
      profile = default
      type = Terminal
    [[[terminal7]]]
      order = 1
      parent = child5
      profile = default
      type = Terminal
  [[single]]
    [[[child0]]]
      type = Window
      parent = ""
      order = 0
      position = 516:187
      maximised = False
      fullscreen = False
      size = 642, 426
      title = /bin/bash
      last_active_term = 3341f869-955d-453e-855d-6369a8323d5f
      last_active_window = True
    [[[terminal1]]]
      type = Terminal
      parent = child0
      order = 0
      profile = single
      uuid = 3341f869-955d-453e-855d-6369a8323d5f
[plugins]
```

</details>

**Comment by mattrose** [comment]
This is fixed in master as part on #724.  I'll create a release soon and I'll try to update the PPA.   In the meantime, as a workaround you can remove the following line `layout_launcher = None` and add a keybinding for the layout_launcher and it will allow you to get the context menu back

**Comment by Juerd** [comment]
Thanks!
